configs/gtp-base.cfg

```diff
@@ -0,0 +1,25 @@
+@include compute/1gpu.cfg
+
+# We're not really interested in detailed logging when running with GTP
+logSearchInfo = false
+logAllGTPCommunication = false
+
+# For compatibility with other engines, we need to set multiStoneSuicideLegal, which
+# means we also need to set the other rules instead of relying on rules = tromp-taylor
+koRule = POSITIONAL
+scoringRule = AREA
+multiStoneSuicideLegal = false
+
+allowResignation = false
+resignConsecTurns = 3
```

Comment:
Do we need to specify this and resignThreshold if we've turned off resignation?

Reply:
I don't think we do. On the other hand, maybe we should actually turn resignation back on if we're leaving it on in `match` and it's the default setting. We should do evaluations with `allowResignation=false` as well but seems reasonable to just use the upstream default as our default unless there's a good reason to change it.

Reply:
>  We should do evaluations with allowResignation=false as well but seems reasonable to just use the upstream default as our default unless there's a good reason to change it.

Per discussion in match I lean towards turning resignations off but I'll ask lightvector to see if has thoughts here.